### PR TITLE
Add set(value) for each interface var.

### PR DESCRIPTION
**Pull request by epabst**
This fixes https://github.com/bashor/ts2kt/issues/21

### CODE REVIEW

File: src/TypeScriptToKotlinWalker.kt [hunk 1]
```diff
@@ -43,9 +43,13 @@ private val COMPARE_BY_NAME = { a: Named, b: Named -> a.name == b.name }
 private val IS_NATIVE_ANNOTATION = { a: Annotation -> a.name == NATIVE }
 
 enum class ImplementationType(private val string: String) {
-    UNSPECIFIED(""), NO_IMPL(ts2kt.kotlin.ast.EQ_NO_IMPL), NO_IMPL_GETTER(PROPERTY_GETTER + EQ_NO_IMPL);
+    UNSPECIFIED(""), NO_IMPL(ts2kt.kotlin.ast.EQ_NO_IMPL), NO_IMPL_GETTER(PROPERTY_GETTER + EQ_NO_IMPL) {
```
Comment (bashor) [inline src/TypeScriptToKotlinWalker.kt]: let's 
* rename it to NO_IMPL_PROPERTY and reformat
* replace`ts2kt.kotlin.ast.EQ_NO_IMPL` with `EQ_NO_IMPL`

File: testData/extendExternalDeclarations/simple.d.kt [hunk 2]
```diff
@@ -3,17 +3,17 @@ package simple
 @native
 fun JQuery.foo(): Unit = noImpl
 @native
-var JQuery.bar: Any get() = noImpl
+var JQuery.bar: Any get() = noImpl; set(value) = noImpl
```
Comment (epabst) [inline testData/extendExternalDeclarations/simple.d.kt]: Fixed.

File: src/TypeScriptToKotlinWalker.kt (hunk 1 above)
Comment (epabst) [inline src/TypeScriptToKotlinWalker.kt]: I decided to do a refactor and actually switch back from ImplementationType to needsNoImpl, and add isInInterface.  This made it much easier to work with, and closer to your original implementation.

Comment (epabst) [inline src/TypeScriptToKotlinWalker.kt]: Updated.

**Comment by bashor** [comment]
Thanks, @epabst!

**Pull request merged**